chart/Chart.yaml

```diff
@@ -6,7 +6,7 @@ type: application
 
 version: 0.7.0
 appVersion: 0.7.0
-kubeVersion: ">= 1.19.0 < 1.22.0"
+kubeVersion: ">= 1.21.0 < 1.24.0"
```

Comment:
Do we need to lock the upper bound?

Reply:
It's a safe upper limit. For limiting users from not letting using the versions which we haven't tested. But I will remove it and see if users report issues with newer versions if they face any.